app/common/data/interfaces/collections.py

```diff
@@ -44,6 +46,10 @@ def create_collection(*, name: str, user: User, grant: Grant, version: int = 1)
     except IntegrityError as e:
         db.session.rollback()
         raise DuplicateValueError(e) from e
+
+    # All collections must have at least 1 section; we provide a default to get started with.
+    create_section(title=DEFAULT_SECTION_NAME, collection=collection)
```

Comment:
One small 🤏  thing that might catch some future regressions would be a little assertion on the create collection interface integration test that makes sure that a default section has been added and is in the right shape?